docs/guides/deploy/k8s_metrics.md

```
@@ -127,7 +127,28 @@ spec:
     matchNames:
-      - dynamo
+      - $NAMESPACE
+```
+
+If you are using planner, you can also create a PodMonitor for the planner:
+```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: dynamo-planner-metrics
+  namespace: $NAMESPACE
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)